web/res/js/membershipForm.js

```diff
@@ -162,7 +162,8 @@ $( function () {
 						'BEZ': 'Lastschrift',
 						'UEB': 'Überweisung',
 						'MCP': 'Kreditkarte',
-						'PPL': 'PayPal'
+						'PPL': 'PayPal',
+						'SUB': 'Sofortüberweisung'
```

Comment:
Would love to lose what is without value, but need your knowledge and gut feeling to make the call. I'd have to understand the whole module (unrelated to this story) before going about and nuking this list. Chances are you are right, because the membership form does only show direct debit, irrespective of this list.